doc/src/base/c.md

```diff
@@ -23,6 +23,7 @@ Base.systemerror
 Base.windowserror
 Core.Ptr
 Core.Ref
+Base.isassigned(::Base.RefValue)
```

Comment:
I added this to `c.md` next to where `Ref` is introduced.. Is this the right place to insert this in the docs? Do we need additional docs on `Base.RefValue` ?